src/TLuaInterpreterMudletObjects.cpp

```diff
@@ -848,15 +848,26 @@ int TLuaInterpreter::getScript(lua_State* L)
 // Documentation: https://wiki.mudlet.org/w/Manual:Lua_Functions#invokeFileDialog
 int TLuaInterpreter::invokeFileDialog(lua_State* L)
 {
+    const int n = lua_gettop(L);
+    QString location = QDir::currentPath();
```

Comment:
Is that the best directory to start with (the location of the executable unless it has been previously changed) - I would have thought the profile's own one would be a more consistent place across all OSs and execution environments?

Reply:
I agree.  This was the unchanged default location since function inception.

Reply:
@vadi2 So would it be helpful to change this to:
```suggestion
    Host& host = getHostFromLua(L);
    const QString location = mudlet::getMudletPath(mudlet::profileHomePath, host.getName());
```

It might *presently* surprise some users who will no longer have to navigate away from:
`C:\Users\`userName`\AppData\Local\Mudlet_64_-PTB\app-4.18.5\`
to:
`C:\Users\`userName`\mudlet-data\`profileName`\` to find the stuff for that profile! :grinning:

Reply:
Sure!